libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -170,8 +170,10 @@ export class WorkspaceInstallManager extends BaseInstallManager {
         // there is a situation that the subspace previously has injected dependencies but removed
         // so we can check if the repoState up to date
         if (repoState.packageJsonInjectedDependenciesHash !== undefined) {
-          shrinkwrapWarnings.push(`Some injected dependencies' package.json might have been modified.`);
-          shrinkwrapIsUpToDate = false;
+          shrinkwrapWarnings.push(
```

Comment:
should the error message light up which repo-state.json contains outdated field of `packageJsonInjectedDependenciesHash`?
Another helpful message is tell user to delete the repo-state.json and try to regenerate again by running `rush update`